fs/fs.go

```diff
@@ -427,8 +427,12 @@ func (fs *filesystem) Mount(ctx context.Context, mountpoint string, labels map[s
 		for _, s := range src {
 			sociDesc, ok := c.imageLayerToSociDesc[s.Target.Digest.String()]
 			if !ok {
-				rErr = fmt.Errorf("unable to resolve layer %q from %q, mounting entire layer using overlayfs: %w", s.Target.Digest, s.Name, snapshot.ErrNoZtoc)
-				continue
+				log.G(ctx).WithFields(logrus.Fields{
+					"layerDigest": s.Target.Digest.String(),
+					"image":       s.Name.String(),
+				}).Infof("skipping mounting layer as FUSE mount: %v", snapshot.ErrNoZtoc)
+				rErr = fmt.Errorf("skipping mounting layer %s as FUSE mount: %w", s.Target.Digest.String(), snapshot.ErrNoZtoc)
+				break
```

Comment:
Why changed to ``break``?

Reply:
If a layer does not have a corresponding ztoc, it does not make sense to continue looking at other sources. So breaking out and returning early.